test/e2e-test/rollout_plan_test.go

```diff
@@ -570,12 +593,10 @@ var _ = Describe("Cloneset based rollout tests", func() {
 				appRollout.Spec.RolloutPlan.BatchPartition = nil
 				return k8sClient.Update(ctx, &appRollout)
 			}, time.Second*15, time.Millisecond*500).Should(Succeed())
-
+		time.Sleep(5 * time.Second)
```

Comment:
why sleep here? Maybe use eventually?

Reply:
that's from rfc-approllout branch. And it's still WIP. We will clean them up later.